db/controllers/like-controller.js

```diff
@@ -76,7 +77,16 @@ likeRouter.get('/liked-albums/:userId', async (req, res) => {
                     model: Like,
                     where: {UserId: userId}
                 }
-            ]
+            ],
+            attributes: [
+                "id",
+                "spotifyId",
+                "image",
+                "name",
+                "artist",
+                [sequelize.fn("COUNT", sequelize.col("Likes.AlbumId")), "likesCount"],
```

Comment:
Including the likesCount for users' liked albums

Reply:
cool!